Crypto/src/X509Certificate.cpp

```diff
@@ -215,12 +221,44 @@ void X509Certificate::init()
 }
 
 
+void X509Certificate::setVersion(long ver)
+{
+	X509_set_version(_pCert, ver - 1);
```

Comment:
We should not set this indiscriminately. In fact, I'm not even sure this should be exposed to end-user - what is the benefit of allowing user to set this, as opposed to us forcing it internally?